sites/partners/src/components/listings/PaperListingDetails/sections/DetailUnits.tsx

```diff
@@ -36,14 +38,22 @@ const DetailUnits = ({ setUnitDrawer }: DetailUnitsProps) => {
     listing.jurisdictions.id
   )
 
+  const enableNonRegulatedListings = doJurisdictionsHaveFeatureFlagOn(
+    FeatureFlagEnum.enableNonRegulatedListings,
+    listing.jurisdictions.id
+  )
+
+  const showNonRegulated =
```

Comment:
suggestion: Can you add a test to DetailUnits.test.tsx where the listing is of type "nonRegulated"?